liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/PackageServerMojo.java

```diff
@@ -35,10 +38,46 @@
 @Mojo(name = "package", defaultPhase = LifecyclePhase.PACKAGE)
 public class PackageServerMojo extends StartDebugMojoSupport {
 
+    public enum PackageFileType {
```

Comment:
This can be private if it is only used in this class?

Reply:
sure